code/modules/paperwork/paper.dm

```diff
@@ -31,6 +31,7 @@
 	var/spam_flag = 0
 	var/contact_poison // Reagent ID to transfer on contact
 	var/contact_poison_volume = 0
+	var/custom = FALSE // Custom paper - ignores checks like reskinning based on text existing
```

Comment:
Just add filled_state/epmty state vars. Seems more useful than something weird like this. (The pamphlets and such could probably use these filled)